vignettes/User-input-example.Rmd

```diff
@@ -213,7 +213,9 @@ plot_activity_heatmap(activity = data_activity, signatures = unique_active_sigs)
 
 ## Active signature interpretation
 We download *Pseudomonas aeruginosa* KEGG pathway terms from the
-[TRIBE](http://tribe.greenelab.com/#/home) webserver.
+[TRIBE](http://tribe.greenelab.com/#/home) webserver. TRIBE also provides Gene
```

Comment:
Oh, also Tribe supports https, so the Tribe url can be changed to ``https://tribe.greenelab.com/#/home`` :smile: